### PR TITLE
Fixing pandas/tests/indexes/multi/test_analytics.py to utilize python…

### PR DESCRIPTION
**Pull request by AncientRickles**
Modified reflect python3 format strings on pandas/tests/indexes/multi/test_analytics.py.  I ran the test and it passed like before.  ref [#29886](https://github.com/pandas-dev/pandas/issues/29886) and [#29547](https://github.com/pandas-dev/pandas/issues/29547).

### CODE REVIEW

File: pandas/tests/indexes/multi/test_analytics.py [hunk 1]
```diff
@@ -330,13 +330,13 @@ def test_numpy_ufuncs(idx, func):
...
         expected_exception = AttributeError
-        msg = "'tuple' object has no attribute '{}'".format(func.__name__)
+        msg = f"'tuple' object has no attribute '{func.__name__}'"
     else:
         expected_exception = TypeError
         msg = (
-            "loop of ufunc does not support argument 0 of type tuple which"
-            " has no callable {} method"
-        ).format(func.__name__)
+            f"loop of ufunc does not support argument 0 of type tuple which"
```
Comment (WillAyd) [inline pandas/tests/indexes/multi/test_analytics.py]:
```suggestion
            "loop of ufunc does not support argument 0 of type tuple which"
```

Shouldn't need the f-prefix unless actually substituting

File: pandas/tests/indexes/multi/test_analytics.py [hunk 2]
```diff
@@ -348,9 +348,9 @@ def test_numpy_ufuncs(idx, func):
 )
 def test_numpy_type_funcs(idx, func):
     msg = (
-        "ufunc '{}' not supported for the input types, and the inputs"
-        " could not be safely coerced to any supported types according to"
-        " the casting rule ''safe''"
-    ).format(func.__name__)
+        f"ufunc '{func.__name__}' not supported for the input types, and the inputs"
+        f" could not be safely coerced to any supported types according to"
```
Comment (WillAyd) [inline pandas/tests/indexes/multi/test_analytics.py]: Same here...

File: pandas/tests/indexes/multi/test_analytics.py [hunk 3]
```diff
@@ -348,9 +348,9 @@ def test_numpy_ufuncs(idx, func):
...
+        f" the casting rule ''safe''"
```
Comment (WillAyd) [inline pandas/tests/indexes/multi/test_analytics.py]: ...and here

**Review by WillAyd — CHANGES_REQUESTED** [review]
Thanks - a few minor cleanups

**Comment by MomIsBestFriend** [comment]
> Fixed to reflect python3 format strings on pandas/tests/indexes/multi/test_analytics.py. I ran the test and it passed like before. Should fix [#29886](https://github.com/pandas-dev/pandas/issues/29886) and [#29547](https://github.com/pandas-dev/pandas/issues/29547).

@AncientRickle can you please replace the word ```fix``` with the word ```ref```?

Just so it wouldn't close the issue.

File: pandas/tests/indexes/multi/test_analytics.py (hunk 3 above)
Comment (AncientRickles) [reply pandas/tests/indexes/multi/test_analytics.py]: Yep, tested it in my python interpreter and it works.  Thanks for the suggestions, @WillAyd

**Comment by jreback** [comment]
thanks @AncientRickles

**Pull request merged**